dcgm/tests/common.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+from datadog_checks.dev import get_docker_hostname, get_here
+
+HOST = get_docker_hostname()
+INSTANCE = {
+    "openmetrics_endpoint": f"http://{HOST}:9400/metrics",
+}
+
+HERE = get_here()
+COMPOSE_FILE = os.path.join(HERE, 'docker', 'docker-compose.yaml')
+
+EXPECTED_METRICS = [
```

Comment:
There are 30 metrics in metadata.csv/metrics.py, but we check only 23 in tests. Is this intentional?

Reply:
(idea: you could manually add them to the fixtures since the file is fake anyway)

Reply:
I think that's a good one, in fact I have faked one metric in the fixture already. One of the metrics from the latest batch that I added.

Reply:
Oh this was a great catch, I found some metrics that weren't named properly as a result of adding them to the tests!!